bench/macro/lsm-tree-bench-wp8.hs

```diff
@@ -382,14 +388,21 @@ main = do
 -- utils: should this be in main lib?
 -------------------------------------------------------------------------------
 
-withSession :: FS.SomeHasFS IO -> FS.FsPath -> (LSM.Session IO -> IO r) -> IO r
-withSession fs path = bracket (LSM.openSession fs path) LSM.closeSession
+withSession ::
+     FS.HasFS IO FsIO.HandleIO
+  -> FS.HasBlockIO IO FsIO.HandleIO
+  -> FS.FsPath
+  -> (LSM.Session IO -> IO r)
+  -> IO r
+withSession hfs hbio path = bracket (LSM.openSession hfs hbio path) LSM.closeSession
 
 defaultTableConfig :: LSM.TableConfig
 defaultTableConfig =  LSM.TableConfig
-    { LSM.tcMaxBufferMemory      = 1000
-    , LSM.tcMaxBloomFilterMemory = 1000
-    , LSM.tcBitPrecision         = ()
+    { LSM.confMergePolicy      = LSM.MergePolicyTiering
+    , LSM.confSizeRatio        = LSM.Four
+    , LSM.confWriteBufferAlloc = 1000
```

Comment:
Picking a default is pretty arbitrary, but that's not even a page. Maybe at least a couple dozen kB?

Reply:
I'll go for 2 MB, which would fill at least 512 pages